tests/xdebug_gc_stats1.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+xdebug: No memleak, return empty runs
```

Comment:
The "xdebug: " prefix is not necessary. I would add "GC Stats: " though instead.